public/app/routes/routes.ts

```diff
@@ -77,7 +76,9 @@ export function setupAngularRoutes($routeProvider: route.IRouteProvider, $locati
       reloadOnSearch: false,
       resolve: {
         component: () =>
-          import(/* webpackChunkName: "SoloPanelPage" */ '../features/dashboard/containers/SoloPanelPage'),
+          SafeDynamicImport(
```

Comment:
Cool stuff @hugohaggmark. Would this work if `SafeDynamicImport` accepted string instead? I assume Webpack would have problems understanding the magic comments.

Reply:
Not sure how those magic comment stuff works in WebPack, could be worth looking at in future PR.

Reply:
Without something like macros I don't think it'd be possible, but maybe it's worth looking at https://github.com/kentcdodds/babel-plugin-macros or https://github.com/LeDDGroup/typescript-transform-macros ?